resources/views/admin/internet/mac_addresses/list.blade.php

```diff
@@ -0,0 +1,125 @@
+<div class="card">
+    <div class="card-header">@lang('internet.admin.manage_register_devices')</div>
+    <div class="card-body">
+        <form action="{{ route('internet.mac_addresses.add') }}" method="post">
+            @if ($errors->any())
+                <div class="alert alert-danger">
+                    <ul>
+                        @foreach ($errors->all() as $error)
+                            <li>{{ $error }}</li>
+                        @endforeach
+                    </ul>
+                </div>
+            @endif
+            <div class="form-row align-items-center">
```

Comment:
Wouldn't it make sense to add the created_at column to the list?

Reply:
And I think a user column would be more important.
The state could be indicated by setting the color of the row, and this would give space for the more important infos.

Reply:
Nice idea, we can do that later.

Reply:
Let's open an issue after this PR is merged.